### PR TITLE
Refactor: configurable file extensions

### LINKED ISSUE #269: Feature: configurable video extensions
It would be nice if people can add additional file extensions, so that sigal would be able to process them as videos. As an example, I'd like to use `.MTS` files that are produced by my camcorder

### PR DESCRIPTION
**Pull request by matelakat**
This change enables people to configure what file extensions should be
recognised as images and videos by setting:

`img_extensions` and `video_extensions`

fixes #269

### CODE REVIEW

File: sigal/templates/sigal.conf.py [hunk 1]
```diff
@@ -49,6 +49,9 @@
 # show_map = False
 # leaflet_provider = 'OpenStreetMap.Mapnik'
 
+# File extensions that should be treated as images
+# img_extensions = ['.jpg', '.jpeg', '.png', '.gif'],
```
Comment (saimn) [inline sigal/templates/sigal.conf.py]: Remove trailing comma

**Comment by saimn** [comment]
Not sure why I did not go this way before, probably to limit the number of settings, but I agree that it's the easiest solution to allow users to cope with unusual extensions.

**Comment by matelakat** [comment]
> Just a minor nitpick

Done @saimn

**Comment by saimn** [comment]
Thanks @matelakat !

**Pull request merged**